video/out/vo_tct.c

```diff
@@ -101,37 +102,24 @@ static int rgb_to_x256(uint8_t r, uint8_t g, uint8_t b)
     return color_err <= gray_err ? 16 + color_index() : 232 + gray_index;
 }
 
-static void print_seq3(struct lut_item *lut, const char* prefix,
+static void print_seq3(bstr *frame, struct lut_item *lut, bstr prefix,
```

Comment:
It seems we have some inconsistency here with regard to using `bstr` vs `struct bstr`. While one is `typedef`-ed to the other, I'm not sure it makes sense to be inconsistent. I recognize that the codebase itself is inconsistent in other places, but maybe we should pick one convention and stick to it?

Reply:
We should use `bstr` everywhere, I used `struct bstr` in priv, because it was nicely aligned with all other struct there, I can revert that.